tendrl/monitoring_integration/upgrades/delete_dashboards.py

```diff
@@ -17,65 +17,72 @@ def delete_dashboards(server_ip, user, password):
+            )
 
     # Deleting the alerts dashboards
     url = "http://%s/grafana/api/orgs/name/Alert_dashboard" \
           % server_ip
-    print ("Getting alerts organization id\n %s \n" % url)
+    utils.print_message("Getting alerts organization id\n %s" % url)
     response = requests.get(url, headers=headers,
                             auth=HTTPBasicAuth(user, password))
     resp = response.json()
 
     if 'id' in resp:
         id = resp['id']
         url = "http://%s/grafana/api/orgs/%s" % (server_ip, id)
-        print ("Deleting alerts organization\n %s" % url)
+        utils.print_message("Deleting alerts organization\n %s" % url)
         response = requests.delete(url, headers=headers,
                                    auth=HTTPBasicAuth(user, password))
         resp = response.json()
         if resp == {u'message': u'Organization deleted'}:
-            print ("Deleted Alert dashboards")
+            utils.print_message("Deleted Alert dashboards")
         else:
-            print ("Failed to delete Alert dashboards %s" % resp)
+            utils.print_message(
+                "Failed to delete Alert dashboards %s" % resp
+            )
 
     else:
-        print ("Failed to delete Alert dashboards.")
-        print ("Organization id not found %s" % resp)
+        utils.print_message(
+            "Failed to delete Alert dashboards."
+            "Organization id not found %s" % resp
+        )
 
 
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("--username", help="grafana admin_user username")
     parser.add_argument("--password", help="grafana admin_user password")
     args = parser.parse_args()
-
     try:
-        print ("\n Modifying brick path separator \n")
-        # Modifying brick path separator | to :
-        os.system(
+        utils.print_message("Stopping carbon-cache service")
+        utils.stop_service("carbon-cache")
+        utils.print_message("Stopping httpd service")
+        utils.stop_service("httpd")
+        utils.print_message("Modifying brick path separator")
+        # Modifying brick path separator from | to :
+        utils.command_exec(
             "find /var/lib/carbon/whisper/tendrl/ -depth  "
             "-type d -name '*|*' -execdir bash -c "
             "'mv \"$1\" \"${1//|/:}\"' bash {} ';'"
         )
-        print ("\n Migrating graphite data \n")
-        os.system(
-            "django-admin migrate --fake dashboard "
-            "--settings=graphite.settings "
-            "--run-syncdb"
-        )
-        os.system(
-            "django-admin migrate --fake-initial "
+        utils.print_message("Removing graphite DB")
+        utils.remove_file("/var/lib/graphite-web/graphite.db")
+        utils.print_message("Initializing graphite DB")
+        utils.command_exec(
+            "django-admin migrate "
             "--settings=graphite.settings "
             "--run-syncdb"
         )
```

Comment:
We should also properly set the ownership of the `graphite.db` to `apache` user and group.
It is true, that execution of *tendrl-ansible* will fix it also, but since the `httpd` service is already started here in the next step, it will be safer to correct the ownership immediately when the database is initialized.

Reply:
I agree to this

Reply:
ok